Pod/Classes/URBNAccordionDataSourceAdapter.h

```diff
@@ -39,7 +41,7 @@ typedef void (^URBNAccordionHeaderViewConfigureBlock) (id view, id object, NSInt
  *  @param identifier (optional)    The reuseIdentifier to be used for this view.  If nil the @viewClass will be used.
  *  @param configurationBlock       The block that configures instances of the view class.
  */
-- (void)registerAccordionHeaderViewClass:(Class)viewClass withIdentifier:(NSString *)identifier withConfigurationBlock:(URBNAccordionHeaderViewConfigureBlock)configurationBlock;
+- (void)registerAccordionHeaderViewClass:(nullable Class)viewClass withIdentifier:(nullable NSString *)identifier withConfigurationBlock:(URBNAccordionHeaderViewConfigureBlock)configurationBlock;
```

Comment:
It's allowed to register a nil class incase you want to register an identifier from a prototypeCell